src/stories/Starting.stories.mdx

```diff
@@ -0,0 +1,196 @@
+import { Meta } from '@storybook/addon-docs';
+
+import github from './assets/github.svg';
+import npm from './assets/npm.svg';
+import figma from './assets/figma.svg';
+
+
+
+<Meta title="Starting" />
+
+<style>
+  {`
+    .subheading {
+      --mediumdark: '#999999';
+      font-weight: 900;
+      font-size: 13px;
+      color: #999;
+      letter-spacing: 6px;
+      line-height: 24px;
+      text-transform: uppercase;
+      margin-bottom: 12px;
+      margin-top: 40px;
+    }
+
+    .link-list {
+      display: grid;
+      grid-template-columns: 1fr;
+      grid-template-rows: 1fr 1fr;
+      row-gap: 10px;
+    }
+
+    @media (min-width: 620px) {
+      .link-list {
+        row-gap: 20px;
+        column-gap: 20px;
+        grid-template-columns: 1fr 1fr;
+      }
+    }
+
+    @media all and (-ms-high-contrast:none) {
+    .link-list {
+        display: -ms-grid;
+        -ms-grid-columns: 1fr 1fr;
+        -ms-grid-rows: 1fr 1fr;
+      }
+    }
+
+    .link-item {
+      display: block;
+      padding: 20px 30px 20px 15px;
+      border: 1px solid #00000010;
+      border-radius: 5px;
+      transition: background 150ms ease-out, border 150ms ease-out, transform 150ms ease-out;
+      color: #333333;
+      display: flex;
+      align-items: flex-start;
+    }
+
+    .link-item:hover {
+      border-color: #1EA7FD50;
+      transform: translate3d(0, -3px, 0);
+      box-shadow: rgba(0, 0, 0, 0.08) 0 3px 10px 0;
+    }
+
+    .link-item:active {
+      border-color: #1EA7FD;
+      transform: translate3d(0, 0, 0);
+    }
+
+    .link-item strong {
+      font-weight: 700;
+      display: block;
+      margin-bottom: 2px;
+    }
+
+    .link-item img {
+      height: 40px;
+      width: 40px;
+      margin-right: 15px;
+      flex: none;
+    }
+
+    .link-item span {
+      font-size: 14px;
+      line-height: 20px;
+    }
+  `}
+</style>
+
+# Getting Started
+
+Developers: This guide will get you up and running with the Popoyoko Design System!
+
+Designers: Please consult the Figma Design System for designer resources.
+
+  <a className="link-item" href="##" target="_blank">
+    <img src={figma} alt="figma" />
+    <span>
+      <strong>Figma</strong>
+      View the design, customize and make a token
+    </span>
+  </a>
+
+## Installation
+
+To install **popoyoko-ui**, you can choose between installing the package directly with [**npm**](), or with [**github**](https://github.com/Popoyoko/popoyoko-ui) (npm being the simplest, I recommend it).
```

Comment:
```bash
[import { Information } from "popoyoko-ui";](github:Popoyoko/popoyoko-ui#package)
```

Reply:
I didn't understand here, when I put the line you put me without characters like this for example: ```bash

it tells me that information hasn't been imported.
Did you want me to put your line below the paragraph?